src/containers/settings/header/Header.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2016, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+import React from 'react';
+
+import styles from './Header.css';
+
+class Header extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+    this.state = { serialNumberExpanded: false };
+  }
+
+  handleClick() {
+    this.setState({ serialNumberExpanded: !this.state.serialNumberExpanded });
```

Comment:
So we should clarify requirements for this. If there isn't an expectation that the expansion to show SN is preserved when you navigate around the different views (e.g., to daily view then back), then handling it with local state like this is fine. If, OTOH, there **is** such an expectation of state preservation, then we should do this via redux. We should get the same clarity around expectations of state preservation of collapsed vs. expanded sections for the Tandem timed settings "profiles" and the different basal profiles for the other pumps. That'll be a larger task to redux-ify, so maybe as a later task to be prioritized.